requirements.txt

```diff
@@ -0,0 +1 @@
+requests==2.18.1
```

Comment:
I had changed this from `>=` because I didn't want to leave it open ended, but I should make all the files consistent.